service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/orderremoval/OrderRemovalTemplate.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.fpl.model.notify.orderremoval;
+
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.fpl.model.notify.NotifyData;
+
+@Builder
+@Data
+public class OrderRemovalTemplate implements NotifyData {
+    private String caseReference;
+    private String caseUrl;
+    private String respondentLastName;
+    private String returnedNote;
```

Comment:
Should be `removalReason`

Reply:
Renamed and also updated in the  SDO/CMO removal notification templates